tests/e2e/performance/test_pvc_attachtime.py

```diff
@@ -106,18 +108,51 @@ def init_full_results(self, full_results):
         return full_results
 
     @pytest.fixture()
-    def pod(self, interface, pod_factory, pvc_factory):
+    def pod_obj_list(
+        self, interface, storageclass_factory, pod_factory, pvc_factory, samples_num
+    ):
         """
-        Prepare pod for the test
+        Prepare sample pods for the test
 
         Returns:
-            pod obj: Pod instance
+            pod obj: List of pod instances
 
         """
         self.interface = interface
-        pvc_obj = pvc_factory(interface=interface, size=self.pvc_size)
-        pod_obj = pod_factory(pvc=pvc_obj)
-        return pod_obj
+        pod_result_list = []
+
+        pvc_size = 5
```

Comment:
can the pvc size also be parameterized? I believe the size doesn't matter for thin PVCs, but the time would change for thick pvcs

Reply:
Size is relevant only for PVC creation (thin vs thick). Time taken to attach a PV to a worker-node should be constant.

Reply:
ah, that is correct. I actually was thinking about PVC creation while making this comment. size is irrelevant.

Reply:
Done